example/src/shared/Wysiwyg/Editor.js

```diff
@@ -3,29 +3,46 @@ import PropTypes from 'prop-types';
 import CodeMirror from 'codemirror';
 
 import EditorWrapper from './EditorWrapper';
+import { EditorAndPreviewWrapper } from './WysiwygStyles';
+import PreviewWysiwyg from './../PreviewWysiwyg/index'
```

Comment:
You don't need the `/index` (node.js will search and load (by default) an index.js file from the folder).
You don't need the `./` (When you import a module from a higher directory)

```js
import PreviewWysiwyg from '../PreviewWysiwyg';
```